Sources/SwiftDocC/Model/DocumentationMarkup.swift

```diff
@@ -74,12 +74,17 @@ struct DocumentationMarkup {
     }
     
     /// Directives which are removed from the markdown content after being parsed.
-    private static let directivesRemovedFromContent = [
+    internal static let directivesRemovedFromContent = [
```

Comment:
nit: `internal` is the default access level
```suggestion
    static let directivesRemovedFromContent = [
```